res/TensorFlowLiteRecipes/BroadcastTo_000/test.rule

```diff
@@ -0,0 +1,5 @@
+# To check if BroadcastTo is Custom op
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "BroadcastTo_EXIST"       $(op_count BROADCAST_TO) '=' 1
```

Comment:
have you check outer rules? what about existing `Custom` op?

Reply:
@seanshpark 

> have you check outer rules? what about existing Custom op?

Oh, yes. I checked the test rule of `Net_BroadcastTo_AddV2_~`.
but, I found out that I have to check the existence of `Custom` after seeing your comment.

I modified the rule code again. Thank you a lot :)